docs/developmentSetup.md

```diff
@@ -53,13 +53,15 @@ sudo chown -R 2000:2000 ./volumes/app/mattermost/
 ```
 8. Now you can start docker `docker-compose up` (add -d flag to start in detached mode). To stop all containers `docker-compose down`.
 
-9. Start c0d3 app `PORT=4000 yarn dev`. If everything went right you will see the landing page. Login with `admin:password`.
+9. Configure your local mattermost server. When you open [http://localhost:8080](http://localhost:8000), you will be greeted with the mattermost login page. Enter any email (this setup won't have an SMTP server, so no actual mails will be sent). First user automatically becomes the server admin. Create new team and enable use of personal access tokens *System Console > Integrations > Custom Integrations*.
 
-10. Configure your local mattermost server. When you open [http://localhost:8080](http://localhost:8080), you will be greeted with the mattermost login page. Enter any email (this setup won't have an SMTP server, so no actual mails will be sent). First user automatically becomes the server admin. Create new team and enable use of personal access tokens *System Console > Integrations > Custom Integrations*.
+10. Generate a personal access token. Close system console, go to team chat (town square) and click on three horizontal bars near your name *Account settings > Security* , then `generate personal access token`. This value will be used as MATTERMOST_ACCESS_TOKEN.
 
-11. Generate a personal access token. Close system console, go to team chat (town square) and click on three horizontal bars near your name *Account settings > Security* , then `generate personal access token`. This value will be used as MATTERMOST_ACCESS_TOKEN.
+11. Keep in mind that by default mattermost requires very secure passwords: at least 10 characters, at least one uppercase letter, one number and one special symbol. Either use such passwords or edit mattermost config at *docker/docker/volumes/app/mattermost/config/config.json* PasswordSettings and restart the server.
```

Comment:
This can be changed in the Mattermost System Console too, might be easier, it's in Authentication > Password.

Might be interesting to provide a pre-configured `config.json` too for convenience.

Reply:
>Might be interesting to provide a pre-configured config.json too for convenience.
I thought about that, but is it really the same config for all users regardless of external environment? E.g. won't my premade config break something for people with macs. I can't test this. 
Probably not since it's the whole point of containers but I am not 100% sure.